src/main/java/kobe/Task.java

```diff
@@ -0,0 +1,138 @@
+package kobe;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+class Task {
+    private boolean done;
+    private String taskName;
+    private String type;
+    private String condition;
+    private LocalDate date;
+    private boolean isConditionLocalDate;
```

Comment:
this attribute name "isConditionLocalDate" is not expressive i think. As i read this, I cannot tell what is the meaning of this attribute. Is it because of the order of the words.